builds/fabric8-che/pom.xml

```diff
@@ -10,7 +10,7 @@
     <parent>
         <artifactId>che-parent</artifactId>
         <groupId>org.eclipse.che</groupId>
-        <version>@che.version@</version>
```

Comment:
< It would be useful for developers to have this PR merged while working on the remaining 
+1, without David's fix it's utterly difficult to develop plugins for rh-che